clockwork_web/static/locales/fr/LC_MESSAGES/messages.po

```diff
@@ -18,56 +18,248 @@ msgstr ""
+
+#: clockwork_web/browser_routes/settings.py:562
+msgid "The requested time_format is unknown."
+msgstr "Le format de date est inconnu."
+
+#: clockwork_web/browser_routes/settings.py:573
+msgid "Missing argument, or wrong format: time_format."
+msgstr "Argument manquant, ou présentant un format incorrect: time_format"
+
+#: clockwork_web/browser_routes/users.py:44
+msgid "Missing argument username."
+msgstr "Le paramètre 'username' est manquant."
+
+#: clockwork_web/browser_routes/users.py:75
+msgid "The requested user has not been found."
+msgstr "L'utilisateur demandé n'a pas pu être trouvé."
+
+#: clockwork_web/core/users_helper.py:125
+msgid "The setting has been updated."
+msgstr "Les paramètres ont été sauvegardés."
+
+#: clockwork_web/core/users_helper.py:129
+msgid "An error occurred."
+msgstr "Une erreur est survenue."
+
+#: clockwork_web/core/users_helper.py:132
+msgid "The provided value is not expected for this setting."
+msgstr "La valeur fournie ne correspond pas au paramètre à modifier."
+
+#: clockwork_web/core/users_helper.py:216
+msgid "The provided value is not expected for this setting"
+msgstr "La valeur fournie ne correspond pas au paramètre à modifier"
+
+#: clockwork_web/core/users_helper.py:245
+msgid "The setting has been reset to its default value."
+msgstr "Le paramètre a été réinitialisé."
+
+#: clockwork_web/templates/base.html:87 clockwork_web/templates/base.html:97
+msgid "Dashboard"
+msgstr "Tableau de bord"
+
+#: clockwork_web/templates/base.html:88
+msgid "Jobs"
+msgstr "Jobs"
+
+#: clockwork_web/templates/base.html:90
+msgid "Clusters"
+msgstr "Grappes de calcul"
+
+#: clockwork_web/templates/base.html:91
+msgid "Settings"
+msgstr "Paramètres"
+
+#: clockwork_web/templates/base.html:94
+msgid "External"
+msgstr "Ressources externes"
+
+#: clockwork_web/templates/base.html:98 clockwork_web/templates/base.html:101
+msgid "Documentation"
+msgstr "Documentation"
+
+#: clockwork_web/templates/base.html:104
+msgid "Admin"
+msgstr "Admin"
+
+#: clockwork_web/templates/base.html:120
+msgid "Logged in as"
+msgstr "Connecté en tant que"
+
+#: clockwork_web/templates/base.html:121
+msgid "Logout?"
+msgstr "Déconnexion"
+
+#: clockwork_web/templates/base.html:142
+msgid "SEARCH"
+msgstr "RECHERCHER"
+
+#: clockwork_web/templates/base.html:157
+msgid "Filter by user"
+msgstr "Filtrer par utilisateur"
+
+#: clockwork_web/templates/base.html:175
+msgid "All users"
+msgstr "Tous les utilisateurs"
+
+#: clockwork_web/templates/base.html:191
+msgid "Only me"
+msgstr "Seulement moi"
+
+#: clockwork_web/templates/base.html:201 clockwork_web/templates/base.html:207
+msgid "Other user..."
+msgstr "Autre utilisateur..."
+
+#: clockwork_web/templates/base.html:218
+msgid "Filter by cluster"
+msgstr "Filtrer par grappe de calcul (/cluster)"
+
+#: clockwork_web/templates/base.html:227
+msgid "Cluster documentation"
+msgstr "Documentation de la grappe de calcul (/du cluster)"
```

Comment:
Unsure where this is used in the interface, but if it messes things up because it's too long, we can use "Documentation grappe". If it's just a popup thing, then we can use "Documentation de la grappe de calcul".